site/guide/manage-inventory-custom-fields.qmd

```diff
@@ -0,0 +1,66 @@
+---
+title: "Manage model inventory custom fields"
+date: last-modified
+---
+
+Create and edit custom fields that appear on all models in your model inventory. 
+
+## Prerequisites
+
+To configure templates, you must hold the `Customer Admin` role. 
+
+## Add or edit custom fields
+
+1. [Log in to the ValidMind UI]({{< var vm_url >}}).
+
+2. Head to your [**Settings**]({{< var vm_url >}}/settings).
+
+3. Under Workspace Settings, click on **Custom Fields**. Here you can edit existing custom fields, or add a new one:
+
+    - To add a new field, click **+ Add new field**. 
+    - To edit an existing field, select **{{< fa pencil >}} Edit** under the Actions column for the desired field.
+
+    At a minimum, you must provide the required information indicated by `*`. You can also choose to indicate that a field is **Required on registration**.
```

Comment:
```suggestion
    At a minimum, you must provide the required information indicated by `*`. You can also choose to indicate that a field is **required on registration**.
```

FYI, weaving references to UI elements like **Required on registration** works in English text, but can break future translation. It might be easier to just make this plain text, bolded for emphasis.